ViacheslavBlazhkov/Lesson15/Lesson15.Homework/Program.cs

```diff
@@ -0,0 +1,105 @@
+﻿namespace Lesson15.Homework
+{
+    class Stack<T>
+    {
+        private T[] stackArray { get; set; }
+
+        public int Count { get; private set; }
+
+        public Stack(T[] array)
+        {
+            stackArray = array;
+            Count = array.Length;
+        }
+
+        public void Push(T item)
+        {
+            T[] array = new T[stackArray.Length+1];
```

Comment:
To not recreate an array every time, you can have a bigger array than the elements you have and extend it only in overfill